### ISSUE TITLE
Model page stays empty with pre 2.0 model data: Error: "neckstimate() requires a valid measurement name as second parameter. (received underBust)"

### ISSUE THREAD

**Issue by stsydow** [freesewing/freesewing.org#250]
At https://freesewing.org/models/ztmnh

Here is what my browsers dev console says:
```
Error: "neckstimate() requires a valid measurement name as second parameter. (received underBust)"
    exports index.js:223
    Qu show.js:382
    Qu show.js:376
    React 6
    unstable_runWithPriority scheduler.production.min.js:342
    React 6
    <anonymous> production-app.js:150
react-dom.production.min.js:4306:12
    React 5
    unstable_runWithPriority scheduler.production.min.js:342
    React 4
    unstable_runWithPriority scheduler.production.min.js:342
    React 6
    <anonym> production-app.js:150
```

**Comment by joostdecock** [comment]
Hmm, underbust has no capital B in it. Or shouldn't.
I'll look into it

**Comment by joostdecock** [comment]
The underBust issue has been fixed with:

```
db.models.updateMany({"measurements.underBust":{$exists:true}}, {$rename: {'measurements.underBust': 'measurements.underbust'}})
```

(this was incorrectly migrated from v1 data)

But there's a problem still with measurements for models with breasts, because the `utils` packages doesn't know about that measurements.

We need to roll out v2.1.3 to fix that, which I hope to do tomorrow as I don't have the time for it right now.

**Comment by joostdecock** [comment]
Fixed in v2.1.3 which is currently being deployed
